pkg/printer/printer.go

```diff
@@ -34,6 +34,13 @@ type StdPrinter struct {
 	Out io.Writer
 }
 
+// Println outputs info to console directly.
+func (sp *StdPrinter) Print(msg string) {
+	if sp.Out != nil {
+		fmt.Fprint(sp.Out, msg)
+	}
```

Comment:
What will happen if `sp.Out` is not nil?

Reply:
If `sp.Out` is not nil, the msg will be written into `sp.Out`.
Otherwise, nothing is printed.